sql_generators/glean_usage/templates/events_first_seen_v1.metadata.yaml

```diff
@@ -0,0 +1,37 @@
+friendly_name: Events First Seen
+description: |-
+  Captures the earliest date that we observe an event for a particular client in the events_stream_v1 table.
+owners:
+  - kbammarito@mozilla.com
+  - vsabino@mozilla.com
+labels:
+  schedule: daily
+  table_type: client_level
+scheduling:
+  dag_name: bqetl_glean_usage
+  task_group: {{ app_name }}
+  date_partition_parameter: null
+  destination_table: >-
+    {% raw %}events_first_seen_v1${{
+    (execution_date - macros.timedelta(hours=24)).strftime("%Y%m%d")
+    }}{% endraw %}
```

Comment:
```suggestion
```
The query updates the entire table (because it joins "current" and "previous"), so we don't need to specify a partition here